src/handlers/pricing/action.ts

```diff
@@ -44,3 +50,8 @@ export const pricingLabelLogic = async (): Promise<void> => {
     logger.info(`Skipping action...`);
   }
 };
+
+export const isParentIssue = (body: string) => {
+  const parentPattern = /-\s+\[( |x)\]\s+#\d+/;
+  return body.match(parentPattern);
+};
```

Comment:
The regex seems limited to the pattern `- [ ] Subtask1`. once it is converted into an issue, it changes like `- https://github.com/ubiquity/ubquibot/issues/1`.

would you take a look again at my concern?

Reply:
Well, actually it it isn't. I just points out the issue number like #issue_number
![image](https://github.com/ubiquity/ubiquibot/assets/119157793/13cc5ed4-09c3-44f1-bc99-eefcb1fbb8fd)
![image](https://github.com/ubiquity/ubiquibot/assets/119157793/7abff099-8fad-416e-97ac-b10d8444d6b8)